cmd/cluster-agent/api/v1/install.go

```diff
@@ -0,0 +1,153 @@
+package v1
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	apiutil "github.com/DataDog/datadog-agent/pkg/api/util"
+	as "github.com/DataDog/datadog-agent/pkg/util/kubernetes/apiserver"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"github.com/gorilla/mux"
+)
+
+// eventChecks are checks that send events and are supported by the DCA
+var eventChecks = []string{
+	"kubernetes",
+}
+
+// Install registers v1 API endpoints
+func Install(r *mux.Router) {
+	r.HandleFunc("/metadata/{nodeName}/{ns}/{podName}", getPodMetadata).Methods("GET")
+	r.HandleFunc("/metadata/{nodeName}", getNodeMetadata).Methods("GET")
+	r.HandleFunc("/metadata", getAllMetadata).Methods("GET")
+	r.HandleFunc("/events/{check}", getCheckLatestEvents).Methods("GET")
```

Comment:
I changed this path from "/{check}/events" to "/events/{check}". This endpoint is currently not used by anything though.